win32ss/user/ntuser/window.c

```diff
@@ -2133,7 +2162,8 @@ PWND FASTCALL IntCreateWindow(CREATESTRUCTW* Cs,
 
 /*
  * @implemented
- * Win: xxxCreateWindowEx
+ * Win: xxxCreateWindowEx(Cs->dwExStyle, ***, ClassName, WindowName, Cs->style,
```

Comment:
Not sure how that's relevant to the PR. This is more pertinent for one own's personal "reversing notes" than being part of our source code...

Reply:
Deleted.